src/bindings/jna/libelektra4j/src/main/java/org/libelektra/PluginLoader.java

```diff
@@ -0,0 +1,40 @@
+package org.libelektra;
+
+import org.libelektra.exception.InstallationException;
+import org.libelektra.plugin.Echo;
+import org.libelektra.plugin.NativePlugin;
+import org.libelektra.plugin.PropertiesStorage;
+import org.libelektra.plugin.Return;
+
+/**
+ * This class can be used to load Plugins from Elektra.
+ * It also loads self implemented Java plugins.
+ */
+public class PluginLoader {
+
+	private Key errorKey;
+
+	public PluginLoader(Key errorKey) {
+		this.errorKey = errorKey;
+	}
+
+	/**
+	 * This plugin loads a Java/Native Elektra Plugin.
+	 *
+	 * @param name the plugin name
+	 * @return the Plugin
+	 * @throws InstallationException if the plugin does not exist
+	 */
+	public Plugin loadPlugin(String name) throws InstallationException {
+		if (name.equals(new Echo().getName())) {
```

Comment:
I am not sure we need direct instantiation of Java plugins. But if we do it should be done in a proper way. The current implementation instantiated every plugin, even if for none of them the name matches.